fobi_custom/plugins/form_elements/fields/types.py

```diff
@@ -45,3 +45,17 @@
                    INTERCEPT_TYPES + FREE_RESPONSE_INTERCEPT_TYPES)
 # All types that cannot be displayed as charts.
 ALL_INVALID_TYPES = list(set(ALL_TYPES) - set(ALL_VALID_TYPES))
+
+# A list of types with their corresponding ACS variables.
+TYPES_TO_ACS_VARIABLES = {
+    'age_observational': 'age_and_sex',
+    'gender_observational': 'age_and_sex',
+    'age_intercept': 'age_and_sex',
+    'gender_intercept': 'age_and_sex',
+    'income': 'household_income',
+    'education': 'educational_attainment',
+    'race': 'race',
+    'employment': 'employment_status',
+    'household_tenure': 'household_tenure',
+    'transportation': 'means_of_transit'
+}
```

Comment:
This map helps us determine which types of survey questions can be associated with ACS data in the DAD.

It's possible we could store this information somewhere on a model, perhaps `SurveyComponent` or somewhere in `django-pldp`? I just couldn't really figure out what the right level of abstraction was.

Reply:
This makes sense to me!